README.md

```diff
@@ -19,27 +19,29 @@ This repo hosts a collection of [standards](./interop) to aid in client interope
+ 26 | Thu, Oct 24, 2019 14:00 UTC      | [agenda](https://github.com/ethereum/eth2.0-pm/issues/89) \| [notes](eth2.0-implementers-calls/call_026.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/dmgoqf/live_eth20_implementers_call_26_20191024_1400_gmt/) | [video](https://www.youtube.com/watch?v=DXGeC7cg71Y) |
+ 27 | Thu, Nov 7, 2019 14:00 UTC      | [agenda](https://github.com/ethereum/eth2.0-pm/issues/95) \| [notes](eth2.0-implementers-calls/call_027.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/dsxbhc/live_eth20_call_27_2019117_1400_gmt/) | [video](https://www.youtube.com/watch?v=4_EGNG-Yek4) |
+ 28 | Thu, Nov 21, 2019 14:00 UTC      | [agenda](https://github.com/ethereum/eth2.0-pm/issues/101) \| [notes](eth2.0-implementers-calls/call_028.md) \| no reddit | [video](https://www.youtube.com/watch?v=DzLrxuN55VA) |
+ 29 | Thu, Dec 5, 2019 14:00 UTC     | [agenda](https://github.com/ethereum/eth2.0-pm/issues/108) \| [notes](eth2.0-implementers-calls/call_029.md) \| no reddit | [video](https://www.youtube.com/watch?v=MxeEWmEdb5E) |
+ 30 | Thu, Dec 19, 2019 14:00 UTC      | [agenda](https://github.com/ethereum/eth2.0-pm/issues/112) \| [notes](eth2.0-implementers-calls/call_030.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/ecrgfh/live_eth20_call_30_20191219_1400_gmt/) | [video](https://www.youtube.com/watch?v=LYLiqpj-wiE) |
+ 31 | Thu, Jan 09, 2020 14:00 UTC      | [agenda](https://github.com/ethereum/eth2.0-pm/issues/118) \| [notes](eth2.0-implementers-calls/call_031.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/em7erh/live_ethereum_20_call_31_202019_1400_gmt/) | [video](https://www.youtube.com/watch?v=u2w4EO9YepI) |
+ 32 | Thu, Jan 23, 2020 14:00 UTC      | [agenda](https://github.com/ethereum/eth2.0-pm/issues/123) \| [notes](eth2.0-implementers-calls/call_032.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/esq90y/live_ethereum_20_call_32_2020123_1400_gmt/) | [video](https://www.youtube.com/watch?v=kt59-FEeWTI) |
+ 33 | Thu Feb 06, 2020 14:00 UTC    | [agenda](https://github.com/ethereum/eth2.0-pm/issues/126) \| [notes](eth2.0-implementers-calls/call_033.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/ezpqyy/live_ethereum_20_call_33_202026_1400_gmt/) | [video](https://www.youtube.com/watch?v=c8BhhPfdy0A) |
+ 34 | Thu, Feb 27, 2020 14:00 UTC   | [agenda](https://github.com/ethereum/eth2.0-pm/issues/129) \| [notes](eth2.0-implementers-calls/call_034.md) \| [reddit](https://www.reddit.com/r/ethereum/comments/fa957t/live_ethereum_20_call_34_2020227_1400_gmt/) | [video](https://www.youtube.com/watch?v=tLiMgFoG_vs) |
+ 35 | Thu, Mar 3, 2020 14:00 UTC  | [agenda](https://github.com/ethereum/eth2.0-pm/issues/132) \| [notes](eth2.0-implementers-calls/call_035.md) \| no reddit | [video](https://www.youtube.com/watch?v=orVYfqP_YuQ) |
```

Comment:
There actually is a reddit -- https://www.reddit.com/r/ethereum/comments/fhl3d5/ethereum_20_call_34_2020312/

it was mislabeled as 34 but was for call 35

Reply:
Ok, updated. 8de9aea